src/Const/README.md

```diff
@@ -0,0 +1,217 @@
+---
+title: "Const"
+description: "Const Monoid"
+layout: "guide"
+weight: 80
+---
+
+```haskell
+Const c a
+```
+`Const` is a function that takes a single value and returns a `Product` type
+that ignores its right side. 
+`Const` is a `chainable` that will take any value `c` and regardless of the 
+method called on it will return a new `Const` with the value `c`.
+
+### Example
+```javascript
+import Const from 'crocks/Const'
+import Pair from 'crocks/Pair'
+import compose from 'crocks/helpers/compose'
+import extend from 'crocks/pointfree/extend'
+import fst from 'crocks/Pair/fst'
+import valueOf from 'crocks/pointfree/valueOf'
+
+Const('Hello World')
+//=> Const 'Hello World'
+
+const days = [ 'Today', 'Tomorrow', 'Yesterday' ]
+
+days.map(Const).reduce((acc, c) => acc.concat(c))
+//=> Const 'Today'
+
+Const(100)
+  .concat(Const(10))
+//=> Const 100
+
+// toLower :: String -> String
+const toLower =
+  x => x.toLowerCase()
+
+// Field :: Pair (Const a) a
+// updateField :: a -> Field
+const updateField =
+  value => Pair(Const(value), value)
+
+// updateField :: Field -> Field
+const resetField =
+  extend(compose(valueOf, fst))
+
+const changed =
+  updateField('Joey')
+    .map(toLower)
+    .chain(updateField)
+//=> Pair( Const "Joey", "joey" )
+
+resetField(changed)
+//=> Pair( Const "Joey", "Joey" )
+
+```
+<article id="topic-implements">
```

Comment:
I was going to say that these are only for the `docs`...but it does not hurt to have them here...So nm. I will take a pass over the READMES to add these, makes it easier to maintain, the more that these two area have in common.